### ISSUE TITLE
Update Results Post Workers Analysis

### ISSUE THREAD

**Issue by Kolano** [PUNCH-Cyber/stoq#96]
**Describe the solution you'd like**
I'd like to be able to update "results" with summary analysis.

**Describe alternatives you've considered**
I had though I could do so with a Decorator plug-in, but have realized that they don't allow updating results, instead outputting an additional "decorators" section.

Closest I can come is outputing a copy of the updated results...
`return DecoratorResponse(response.results)`
...though that then requires clean-up to purge the duplicate data in later processing.

**Comment by mlaferrera** [comment]
This is more or less what a `DecoratorPlugin` is designed to do, take the collective of the results and do a post-analysis analysis of them. If you only want the decorator results saved, you'll need to either   create a new decorator plugin or update one to save just the decorator results. Either way, the work flow you are looking for can be done with plugins alone.

As for the error above, it's hard to tell exactly what caused this, but it appears as though you have inserted something other than a dict into the decorator results. As identified [here](https://github.com/PUNCH-Cyber/stoq/blob/master/stoq/data_classes.py#L383-L395), the `DecoratorResponse.results` must be a dict.

**Comment by Kolano** [comment]
I have created a new decorator, but it seemed writing out results was the responsibility of connector plug-ins not a decorator. For now I'll just clean up the duplicated results when I load them to ElasticSearch. I had expected a decorator to revise the original content though, rather than merely appending separate details.

Sorry I removed the error details as I had accounted for it before your response, but yes. I believe they were caused by trying to return the full response object rather than response.results.
